doc/modules/density.rst

```diff
@@ -18,6 +18,8 @@ an unsupervised clustering scheme.
 Density estimation is a very simple concept, and most people are already
 familiar with one common density estimation technique: the histogram.
 
+.. _histogram:
+
```

Comment:
I think we can remove this for now, since we are not using the reference anymore.